.teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt

```diff
@@ -9,7 +9,7 @@ import model.Stage
 open class BaseGradleBuildType(model: CIBuildModel, val stage: Stage? = null, usesParentBuildCache: Boolean = false, init: BaseGradleBuildType.() -> Unit = {}) : BuildType() {
 
     val buildCache: BuildCache =
-        if (stage != null && stage.disablesBuildCache) NoBuildCache
+        if (stage != null) NoBuildCache
```

Comment:
This is bad, since now we will always disable the build cache if a stage is used. Better remove that line completely.

Reply:
Removed.